src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignmentRegion.java

```diff
@@ -0,0 +1,169 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import com.github.lindenb.jbwa.jni.AlnRgn;
+import htsjdk.samtools.Cigar;
+import htsjdk.samtools.CigarElement;
+import htsjdk.samtools.CigarOperator;
+import htsjdk.samtools.TextCigarCodec;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.read.CigarUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.util.List;
+import java.util.Objects;
+
+class AlignmentRegion {
+
+    final String contigId;
+    final String breakpointId;
+    final Cigar cigar;
```

Comment:
It looks like that the 3-parameter ctor and the actor taking a read would put the cigar into a forward strand cigar. So I think it makes sense to change the field name here to something like `forwardStrandCigar`, because it is exposed to other classes in this package.